tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Backward compatibility and test vector generation
+
+Crate `prio` supports Prio v2, used in [Exposure Notification Private Analytics](enpa-whitepaper), as implemented in [`prio-server`](https://github.com/abetterinternet/prio-server), as well as the Prio v3 Verifiable Distributed Aggregation Function used in [Privacy Preserving Measurements](ppm-spec).
+
+To ensure backward compatibility with older versions of crate `prio` and [other implementations of Prio v2](https://github.com/google/libprio-cc), we continuously test `prio`'s Prio v2 implementation against a test vector recorded with `prio` 0.4.0. That test is implemented in `tests/backward_compatibility.rs`. The test vector was generated using `tests/accumulating.rs`. To generate a new set of test vectors, run:
+
+    cargo test --test accumulating -- --nocapture
+
+...and the test will print base64 inputs and an expected sum to stdout, which may then be plugged into a backward compatibility test as illustrated in `tests/backward_compatibility.rs`. Adjust `number_of_clients` to generate longer or shorter test vectors.
```

Comment:
Best practice would be to dump a JSON blob, which can easily be consumed by multiple test vector runners. In Go I would put the JSON blob in a directory called `testdata/` located in the module with the test vector runners. I'm not sure what the right thing would be to do for Rust, but I think it makes perfect sense to just stick it in `tests/testdata`.